app/src/main/res/values/strings.xml

```diff
@@ -63,6 +63,7 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="last_modified">Last modified</string>
     <string name="select_this_folder">Select this folder</string>
     <string name="select">Select</string>
+    <string name="select_all">Select all</string>
```

Comment:
IMO the button title should change to 'clear' if all items are selected.

Reply:
Better yet we can do the following:

1. If no items selected make the text `Select all`
2. If one or more items selected make the text `Clear (n)` where n is the number of items selected

Then we don't need to modify the title to add a clear button etc. Changes become much simpler